README.md

```diff
@@ -28,6 +28,7 @@ jobs:
           errorMessage: ':no_entry: PR has more than **{allowed} additions**. Split it into smaller PRs.' # optional
           warningSize: 300 # optional
           warningMessage: ':warning: PR has more than **{allowed} additions**. Consider splitting it into smaller PRs.' # optional
+          comment: false # to skip comment creation in the PR, optional
```

Comment:
Let's rather control it with the `errorMessage`/`warningMessage` parameters: if they are empty, comment should not be added. 

First, they can be controlled independently in that case. Second, only the single source of truth remains: either comment is empty and it should not be added, or comment is not empty and it should be added.

Reply:
"if they are empty" do you mean a config like this?

```
  pr-size:
    runs-on: ubuntu-latest
    name: PR size
    steps:
      - name: Checkout
        uses: actions/checkout@v3
        with:

      - uses: ookami-kb/gh-pr-size-watcher@v1
        with:
          githubToken: ${{ secrets.GITHUB_TOKEN }}
          errorSize: 1000
          warningMessage: ""
          errorMessage: ""
          excludePaths: |
            pnpm-lock.yaml
```

Reply:
yeah, exactly

Reply:
Ok, we can close this PR

Reply:
I think this PR makes sense, as it doesn't work like that for now. If you update it to use this logic, it would be great.